components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from '@apollo/client/react/components';
+import { useRouter } from 'next/router';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { fontSize, margin } from 'styled-system';
+
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import NextIllustration from './home/HomeNextIllustration';
```

Comment:
Why use this rather than `components/Image`? If `HomeNextIllustration` is a generic component meant to be used elsewhere, it should not be in the `home` subfolder

Reply:
Ah, yes. It's a mistake; I was somehow thinking it could be used everywhere. 😄

Reply:
Ops, missed that somehow. 😮 Sorry, I'll update it.